sycl/tools/sycl-vxx/bin/sycl_vxx.py

```diff
@@ -238,81 +345,135 @@ def _downgrade(self):
         kernel_prop_opt = ["-kernelPropGen",
                            "--sycl-kernel-propgen-output", f"{kernel_prop}"]
 
-        kernel_prop_opt.extend(self.vitis_version.get_kernel_prop_opt())
-
         opt_options = [
             "--lower-delayed-sycl-metadata", "-lower-sycl-metadata",
             "--sycl-vxx", "--sycl-prepare-clearspir", "-S", "-preparesycl",
             *kernel_prop_opt,
             "-globaldce",
             "-strip-debug",
-            self.linked_kernels,
+            inputs,
             "-o", prepared_kernels
         ]
         args = [opt, *opt_options]
-        self._dump_cmd("03-prepare.cmd", args)
+        self._dump_cmd("prepare", args)
         subprocess.run(args, check=True)
         with kernel_prop.open('r') as kp_fp:
             self.kernel_properties = json.load(kp_fp)
         opt_options = ["--sycl-vxx", "-S", "-vxxIRDowngrader"]
-        self.downgraded_ir = (
+        downgraded_ir = (
             self.tmpdir / f"{self.outstem}_kernels-linked.opt.ll")
         args = [
             opt, *opt_options, prepared_kernels,
-            "-o", self.downgraded_ir
+            "-o", downgraded_ir
         ]
-        self._dump_cmd("04-downgrade.cmd", args)
```

Comment:
So now we do not longer have the phase numbering in command names?

Reply:
phase numbers are still there but they are now automatic